eslint.config.js

```diff
@@ -72,5 +72,11 @@ module.exports = {
         Terra: true,
       },
     },
+    {
+      files: ['**/*/terra-dev-site/**/*.jsx'],
+      rules: {
+        'import/no-unresolved': 'off',
```

Comment:
What about 
```
          "import/no-extraneous-dependencies": "off",
          "import/extensions": "off",
```

Reply:
Those rules and the others not included here were primarily (if not exclusively) used with doc files using the DocTemplate, with raw-loader imports and the like. With mdx usage, these overrides aren't necessary, and including them by default might mask actual problems in team's example component files. So I opted to not include them.

Reply:
Once this is merged, can we log issues to each mono-repo to remove our disabled rules from the package.json and associated files?